pages/resources/user-settings.mdx

```diff
@@ -0,0 +1,185 @@
+# User Settings Resource
+
+User settings are options that a user can configure to change the behavior of their account or client.
+
+<Alert type="info">
+  Protobuf user settings are not yet documented here. Refer to [this
+  repository](https://github.com/dolfies/discord-protos) for the data structures.
+</Alert>
```

Comment:
```suggestion
<Alert type="info">

Protobuf user settings are not yet documented here. Refer to [this repository](https://github.com/dolfies/discord-protos) for the data structures.

</Alert>
```
This must be formatted like this or it breaks
![tfqAFFquvF 1](https://github.com/user-attachments/assets/8e73e818-c104-45af-b1ae-937013f86d64)